### PR TITLE
changes done by IBIS stl team

### PR DESCRIPTION
**Pull request by rathisekaran**

### CODE REVIEW

File: anonymizer_methods.py [hunk 1]
```diff
@@ -5,7 +5,12 @@
 import re
 import glob
 import platform
+import errno
 from shutil import move
+
+# hard coded path or Tom
+dcmpath = "/Volumes/botteron/Imaging/Software/DCMTK/dcmtk-3.6.0-mac-i686-dynamic/bin/"
```
Comment (mohadesz) [inline anonymizer_methods.py]: This is a path specific to IBIS and should not be in this file

File: anonymizer_gui.py [hunk 2]
```diff
@@ -174,4 +175,5 @@ def zipDicom(self):
 if __name__ == "__main__":
     root = Tk()
     app = dicom_anonymizer(root)
+    root.lift()
```
Comment (driusan) [inline anonymizer_gui.py]: This one line looks like it does something different from the rest of the pull request and can probably be moved to a different pull request (I think it just raises the window to the top.), to make this one a little easier to read

File: anonymizer_methods.py [hunk 3]
```diff
@@ -30,14 +35,16 @@ def Grep_DICOM_values(dicom_dir, dicom_fields):
...
     # TO DO: Need to check if file is dicom though, otherwise go to next one
     file_list = []
-    for f in os.listdir(dicom_dir):
-        file_list.append(f)
-    dicom_file = dicom_dir + os.path.sep + file_list[1]
+    for dirname, dirnames, filenames in os.walk(dicom_dir):
+        for filename in filenames:
+            dicom_file = os.path.join(dirname, filename)
+            break
+
```
Comment (driusan) [inline anonymizer_methods.py]: This seems weird and I'm not sure what it's trying to do.

Presumably there's a reason to use os.walk instead of os.listdir, but the new logic doesn't look right to me. It has a for loop which always assigns a variable and breaks out unconditionally after the first iteration, and even then the assignment will get clobbered on subsequent iterations of the outer loop?
